packages/ckeditor5-media-embed/src/mediaregistry.js

```diff
@@ -67,6 +67,13 @@ export default class MediaRegistry {
 		 * @member {Array}
 		 */
 		this.providerDefinitions = providerDefinitions;
+
+		/**
+		 * The view element name for newly added media embed.
+		 *
+		 * @member {String}
+		 */
+		this.elementName = config.elementName;
```

Comment:
It looks like `this.elementName` is not used, so it does not need to be added.

Reply:
Done in 84aa3c0b98